apps/src/music/blockly/fields.js

```diff
@@ -59,10 +60,10 @@ export const fieldRestDurationDefinition = {
   type: 'field_dropdown',
   name: FIELD_REST_DURATION_NAME,
   options: [
-    ['\u00bd beat', '0.125'],
-    ['1 beat', '0.25'],
-    ['2 beats', '0.5'],
-    ['1 measure', '1'],
-    ['2 measures', '2'],
+    [`\u00bd ${musicI18n.blockly_fieldRestBeat()}`, '0.125'],
```

Comment:
Sorry for missing this earlier... I suspect the 1/4 character should also be part of the localized string, similar to the other numbers...

Reply:
Oh yes good call, the number characters should probably be localized too

Reply:
Updated to include this character and the number characters